src/components/MovieDetails/index.tsx

```diff
@@ -131,9 +131,14 @@ const MovieDetails: React.FC<MovieDetailsProps> = ({ movie }) => {
     revalidate();
   };
 
-  const movieAttributes = [];
+  const movieAttributes: React.ReactNode[] = [];
 
-  if (data.releases.results.length > 0) {
+  if (
+    data.releases.results.length &&
+    (data.releases.results.find((r) => r.iso_3166_1 === 'US')?.release_dates[0]
```

Comment:
Are these additional `if` conditions necessary?  If so, do we need them in `TvDetails` as well?

Reply:
Also, is it the desired behavior to default to the US release dates?  I'm located in the US and personally don't have an issue with it, but I know we have users all around the globe.

Reply:
Unfortunately yes, I've come across a few instances where the `certification` is just an empty string, but only for movies. It might also be the case for tv shows, so I can add this for tv shows as well.
On TMDB's side, there is now validation that prevents this (empty string) from being added to new shows/movies, so the empty strings are just remnants from before that validation was implemented.